### ISSUE TITLE
3 dots instead of icon in tray since v5.1.3

### ISSUE THREAD

**Issue by mike2307** [vladimiry/ElectronMail#580]
Since the update to the flatpak version of electronmail v5.1.3 I'm no longer having the envelop icon in the tray.
Instead the tray just shows 3 dots instead of the icon.
It's clickable though and appears to be working normally apart from that.
It's just an optical issue

Edit: screenshot
![Screenshot from 2023-01-19 09-08-48](https://user-images.githubusercontent.com/29705161/213388402-4497d31a-848d-42bc-8873-affde81777c8.png)

Please investigate.
Thanks a lot in advance. :)

**Comment by mike2307** [comment]
Just as a sidenote:
I' using gnome-shell-extension-appindicator to show the tray icons.

**Comment by vladimiry** [comment]
I don't think I can do anything to fix it here by some app code change.

Since the recent version, @electron switched to [StatusNotifierItem](https://www.freedesktop.org/wiki/Specifications/StatusNotifierItem/)-based tray icon implementation and so some tray-related issues expected to happen until all the windows managers get decent support for this standard.

Flatpak container itself adds additional issues to the case. I'm subscribed to some of those issue, and they are in the work in progress status.

To recap, you are lucky to get the 3 dots icon vs no icon at all 😀
